app/models/miq_action.rb

```diff
@@ -742,7 +742,7 @@ def action_container_image_annotate_deny_execution(action, rec, inputs)
       return
     end
 
-    unless rec.digest.present?
+    unless !rec.ems_ref.blank?
```

Comment:
unless + ! + blank? is a recipe for confusion...  This is equivalent to `if rec.emd_ref.blank?`